locales/puppetserver.pot

```diff
@@ -252,11 +252,15 @@ msgid "Removed certificate request for {0} at ''{1}''"
 msgstr ""
 
 #: src/clj/puppetlabs/puppetserver/certificate_authority.clj
-msgid "Revoked {0} certificate with serial {1}"
+msgid "Revoked {0} certificate."
```

Comment:
We decided that this was taken care of other automation, right?

Reply:
Yeah the POT file should get overwritten as part of the Jenkins pipeline.